### PR TITLE
[author] [new] adding new pnp scoped client side libraries

### PR DESCRIPTION
**Pull request by patrick-rodgers**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/pnp/pnp
 * Official website (optional, not the repository): https://pnp.github.io/pnp/
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [X] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [X] More than 800 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by patrick-rodgers** [comment]
These libraries are the successors to the sp-pnp-js library (https://github.com/sharepoint/pnp-js-core) already hosted here. It is not yet at the popularity as we have just launched it, but hoping to get things added to cdnjs to assist with folks migrating.

Thanks!

**Comment by sufuf3** [comment]
Some lib have enough popularity recently.
Like https://www.npmjs.com/package/@pnp/common
https://npm-stat.com/charts.html?package=%40pnp%2Fcommon
I'll review this PR in my free time these days.
Thanks.

**Comment by patrick-rodgers** [comment]
Any updates on this? Been open since Feb - anything we can do on our end to help out? Thanks!

**Comment by patrick-rodgers** [comment]
I did recently sync this with master again so it should be easy to merge if you get a chance to review. Please do let me know if I can help out on my side. Thanks for all your hard work on cdnjs!

File: ajax/libs/pnp-common/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pnp-common",
+  "filename": "common.es5.umd.js",
```
Comment (PeterDaveHello) [inline ajax/libs/pnp-common/package.json]: I suppose this should better be `common.es5.umd.min.js`? Same as the other libraries.

**Comment by PeterDaveHello** [comment]
@patrick-rodgers btw, maybe you may wanna link your GitHub account with the email you made the commit?

**Comment by patrick-rodgers** [comment]
@PeterDaveHello - Thanks! I have updated the files to use .min as well rebased to again get tight to master. Looks good on my end, let me know if you need any other changes. Thanks!

**Comment by patrick-rodgers** [comment]
Thanks!

**Pull request merged**